src/components/reservation/ReservationModalBody.tsx

```diff
@@ -113,7 +99,7 @@ const ReservationModalBody = ({
   };
   const { reservations, holdings } = holdingsData[0];
   const { patron } = userData;
-  const authorLine = getAuthorLine(mainManifestation, t);
+  const authorLine = getAuthorLine(selectedManifestations[0], t);
```

Comment:
I guess this is one place where using one of the selected manifestations is appropriate. We are in trouble if the array is empty though 🤔 .